_es/tour/multiple-parameter-lists.md

```diff
@@ -29,10 +52,78 @@ Aquí se muestra un ejemplo:
       println(filter(nums, modN(2)))
       println(filter(nums, modN(3)))
     }
+```
+{% endscalafiddle %}
 
-_Nota: el método `modN` está parcialmente aplicado en las dos llamadas a `filter`; esto significa que solo su primer argumento es realmente aplicado. El término `modN(2)` devuelve una función de tipo `Int => Boolean` y es por eso un posible candidato para el segundo argumento de la función `filter`_
+_Nota: el método `modN` está parcialmente aplicado en las dos llamadas a `filter`; esto significa que solo su primer argumento es realmente aplicado. El término `modN(2)` devuelve una función de tipo `Int => Boolean` y es por eso un posible candidato para el segundo argumento de la función `filter`._
 
 Aquí se muestra la salida del programa anterior:
 
     List(2,4,6,8)
     List(3,6)
+
+
+### Casos de uso
+
+Casos de uso sugeridos para múltiples listas de parámetros incluyen:
+
+#### Inferencia de tipos
+
+En Scala, la inferencia de tipos se realiza parámetro a parámetro.
+Suponer que se dispone del siguiente método:
+
+```tut
+def foldLeft1[A, B](as: List[A], b0: B, op: (B, A) => B) = ???
+```
+
+Y se desea invocarlo de la siguiente manera, pero puede comprobar que no compila:
```

Comment:
Although this sentence is good, I think that is a bit hard to understand and it could be improved somehow. An idea:
"Si se invoca de la siguiente manera, se puede comprobar que no compila correctamente".

Reply:
Me parece acertada la propuesta